sopel/irc/utils.py

```diff
@@ -51,3 +55,13 @@ def nop(bot, cap):
         self.arg = arg
         self.failure = failure or nop
         self.success = success or nop
+
+
+class MyInfo(collections.namedtuple('MyInfo', MYINFO_ARGS)):
+    """Store client, servername, and version from ``RPL_MYINFO`` events.
+
+    .. seealso::
+
+        https://modern.ircdocs.horse/#rplmyinfo-004
+    """
+    # TODO: replace by a class using typing.Namedtuple (new in Python 3.5+)
```

Comment:
```suggestion
    # TODO: replace by a class using typing.NamedTuple (new in Python 3.5+)
    # probably in Sopel 8.0 (due to drop most old Python versions)
```

https://docs.python.org/3/library/typing.html#typing.NamedTuple